File: tests/block_test.py
#!/usr/bin/env python

from parser_test_helper import *


class BlockTest(ParserBaseTest):
	def test_block(self):
		skip()  # old ruby stuff
		assert_result_is(parse("begin 1.times do 1 end end"), 1)

	def test_block_returns(self):
		skip()  #
		assert_result_is(parse("print begin 1.times do 1 end end"), 1)

File: angle.js/test/block.test.js
require('./angle_base_test');

class BlockTest extends (ParserBaseTest) {

	test_block(){
		assert_result_is("{1}", 1)
	}

	test_ruby_block() {
		skip()  // old ruby stuff
		assert_result_is(parse(`begin 1.times do 1 end end`), 1);
	}

	test_ruby_block_returns() {
		skip()  //;
		assert_result_is(parse(`print begin 1.times do 1 end end`), 1);

	}
}
new BlockTest()
